doc/source/whatsnew/v1.0.0.rst

```diff
@@ -28,7 +28,7 @@ Enhancements
 Other enhancements
 ^^^^^^^^^^^^^^^^^^
 
--
+- Implemented :meth:`pandas.core.window.Window.var` and :meth:`pandas.core.window.Window.std` functions (:issue:`26597`)
```

Comment:
These need to be added to `doc/source/reference/window.rst`.

Reply:
were these added?

Reply:
I have added std and var [here](https://github.com/pandas-dev/pandas/pull/27682/files#diff-8c6a93a870d3534c506b9c1a222c879aR38). Is there anything else?